cnames_active.js

```diff
@@ -2184,6 +2184,7 @@ var cnames_active = {
   "orbiton": "orbitonjs.github.io",
   "orga": "orgapp.github.io/orgajs",
   "origami": "origami-cms.github.io/cms",
+  "orpheuslau":"https://github.com/orpheuslau/node-api.git",
```

Comment:
```suggestion
  "orpheuslau": "orpheuslau.github.io/node-api",
```

As the CI error notes, you should be targetting the GitHub Pages site, not the repository itself.